docusaurus/docs/iOS/components/reactions.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Reactions
+---
+
+The Stream Chat API provides built-in support for adding reactions to messages. The component library provides default components to enable reaction selection and display.
+
+## Basic Usage
+
+Message reactions come out of the box with Stream Chat. The SDK will respect your channel configuration, if you disable reactions for a channel or channel type from the dashboard, the SDK will not render the UI for reactions.
+
+## Reactions Picker
+
+When you long-press a message, the SDK will show a reactions picker. The `ChatMessageReactionsVC` view controller allows the user to toggle message reactions. Most of the times, changing the sub-components used by this class or its configurations is enough. For more complex customizations you can sub-class `ChatMessageReactionsVC` and use it. 
+
+```swift
+
+class CustomChatMessageReactionsVC: ChatMessageReactionsVC {}
+
+Components.default.messageReactionsVC = CustomChatMessageReactionsVC.self
+```
+
+### Custom Reactions and Images
+
+The `ChatMessageReactionsVC` picker uses the `components.reactionsBubbleView` component to render the reactions available to the user and to render their state. 
+
+You can change the list of supported message reaction from the `Appearance` object, here is an example on how you can use your own set of reactions
+
+```swift
+let reactionFireSmall: UIImage = UIImage(named: "fireSmall")!
+let reactionFireBig: UIImage = UIImage(named: "fireBig")!
+let reactionWaveSmall: UIImage = UIImage(named: "waveSmall")!
+let reactionWaveBig: UIImage = UIImage(named: "waveBig")!
+
+let customReactions: [MessageReactionType: ChatMessageReactionAppearanceType] = [
+    .init(stringLiteral: "fire"): ChatMessageReactionAppearance(
+        smallIcon: reactionFireSmall,
+        largeIcon: reactionFireBig
+    ),
+    .init(stringLiteral: "wave"): ChatMessageReactionAppearance(
+        smallIcon: reactionWaveSmall,
+        largeIcon: reactionWaveBig
+    )
```

Comment:
```suggestion
    "fire": ChatMessageReactionAppearance(
        smallIcon: reactionFireSmall,
        largeIcon: reactionFireBig
    ),
    "wave": ChatMessageReactionAppearance(
        smallIcon: reactionWaveSmall,
        largeIcon: reactionWaveBig
    )
```
Whenever you see you are using "stringLiteral" as an argument in an initialiser, that is a bad sign. This initialiser comes from `ExpressibleByStringLiteral`, the goal of this is too initialise a type from just a string, but you need to help the compiler understand what type you want. In this case you have `customReactions: [MessageReactionType: ...]` so you only need the string to initialise it. 

Another example:
**This is wrong:**
`let token = Token(stringLiteral: "....")`
**This is good:**
`let token: Token = "..."`

To sum up, we should not use the string literal initialiser ourselves, this is for the compiler to use it. In case the value comes from a property or something else, then it is better to use for example in this case the rawValue initialiser:
`let token = Token(rawValue: remoteConfig.token)`